site/src/comparison.rs

```diff
@@ -231,6 +231,12 @@ pub enum Metric {
     LlvmBitcodeSize,
     #[serde(rename = "size:llvm_ir")]
     LlvmIrSize,
+    /// Total bytes of a generated documentation directory
+    #[serde(rename = "size:doc_bytes")]
+    DocByteSize,
+    /// Number of files inside a generated documentation directory.
+    #[serde(rename = "size:doc_files")]
+    DocFilesSize,
```

Comment:
`DocFilesCount` seems much clearer (especially in isolation)

I think `size:doc_files` should be find if we think `size:doc_file_count` is too long.

Reply:
I agree, that's a better name, I have renamed it.